src/script/calling/Participant.ts

```diff
@@ -18,68 +18,72 @@
 
 export type UserId = string;
 export type ClientId = string;
 
 export class Participant {
   // Video
-  public videoState: ko.Observable<VIDEO_STATE>;
-  public videoStream: ko.Observable<MediaStream | undefined>;
-  public blurredVideoStream = ko.observable<{stream: MediaStream; release: () => void} | undefined>(undefined);
-  public hasActiveVideo: ko.PureComputed<boolean>;
-  public hasPausedVideo: ko.PureComputed<boolean>;
-  public sharesScreen: ko.PureComputed<boolean>;
-  public sharesCamera: ko.PureComputed<boolean>;
-  public startedScreenSharingAt: ko.Observable<number>;
-  public isActivelySpeaking: ko.Observable<boolean>;
-  public isSendingVideo: ko.PureComputed<boolean>;
-  public isAudioEstablished: ko.Observable<boolean>;
+  public readonly videoState = observable(VIDEO_STATE.STOPPED);
+  public readonly videoStream = observable<MediaStream | undefined>();
+  public readonly blurredVideoStream = observable<{stream: MediaStream; release: () => void} | undefined>();
+  public readonly hasActiveVideo: ko.PureComputed<boolean>;
+  public readonly hasPausedVideo: ko.PureComputed<boolean>;
+  public readonly sharesScreen: ko.PureComputed<boolean>;
+  public readonly sharesCamera: ko.PureComputed<boolean>;
+  public readonly startedScreenSharingAt = observable<number>(0);
+  public readonly isActivelySpeaking = observable(false);
+  public readonly isSendingVideo: ko.PureComputed<boolean>;
+  public readonly isAudioEstablished = observable(false);
 
   // Audio
-  public audioStream: ko.Observable<MediaStream | undefined>;
-  public isMuted: ko.Observable<boolean>;
+  public readonly audioStream = observable<MediaStream | undefined>();
+  public readonly isMuted = observable(false);
 
   constructor(
-    public user: User,
-    public clientId: ClientId,
+    public readonly user: User,
+    public readonly clientId: ClientId,
   ) {
-    this.videoState = ko.observable(VIDEO_STATE.STOPPED);
-    this.hasActiveVideo = ko.pureComputed(() => {
+    this.hasActiveVideo = pureComputed(() => {
       return (this.sharesCamera() || this.sharesScreen()) && !!this.videoStream();
     });
-    this.sharesScreen = ko.pureComputed(() => {
+    this.sharesScreen = pureComputed(() => {
       return this.videoState() === VIDEO_STATE.SCREENSHARE;
     });
-    this.sharesCamera = ko.pureComputed(() => {
+    this.sharesCamera = pureComputed(() => {
       return [VIDEO_STATE.STARTED, VIDEO_STATE.PAUSED].includes(this.videoState());
     });
-    this.hasPausedVideo = ko.pureComputed(() => {
+    this.hasPausedVideo = pureComputed(() => {
       return this.videoState() === VIDEO_STATE.PAUSED;
     });
-    this.videoStream = ko.observable();
-    this.audioStream = ko.observable();
-    this.isActivelySpeaking = ko.observable(false);
-    this.startedScreenSharingAt = ko.observable();
-    this.isMuted = ko.observable(false);
-    this.isSendingVideo = ko.pureComputed(() => {
+    this.isSendingVideo = pureComputed(() => {
```

Comment:
Those are only changes to make typescript and eslint happier. Nothing interesting to see there